src/e2e_test/perf_tool/ui/index.html

```
@@ -0,0 +1,1135 @@
+          </div>
+        </div>
+        <div class="separator">OR</div>
+        <div class="section">
+          <div class="gcs-form">
+            <label>GCS Bucket</label>
+            <input type="text" id="gcs-bucket" placeholder="my-bucket-name">
+            <label>Prefix (optional)</label>
+            <input type="text" id="gcs-prefix" placeholder="perf-results">
+            <label>Date filter (optional, YYYY/MM/DD)</label>
+            <input type="text" id="gcs-date" placeholder="2026/04/16">
+            <button class="btn" id="gcs-load-btn">Load from GCS</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)